src/avt/Database/Database/avtGenericDatabase.C

```diff
@@ -7252,6 +7253,11 @@ avtGenericDatabase::CommunicateGhosts(avtGhostDataType ghostType,
             s = CommunicateGhostNodesFromGlobalNodeIds(ds, doms, spec, src);
         else if (canDoStreamingGhosts)  // Zones not Nodes
             s = CommunicateGhostZonesWhileStreaming(ds, doms, spec, src);
+        else
+        {
+            avtGhostNodeGenerator gnf;
+            s = gnf.CreateGhosts(ds);
```

Comment:
Nit: Would be better to just change `bool s = false` to `bool madeGhosts = false` and remove **s** and eliminate the extra assignment of **s** to **madeGhosts** later on.

Reply:
I suspect "s" was introduced to keep the line lengths reasonable. I instead just eliminated "madeGhosts" and used "s" instead.

Reply:
Understandable. It's not hard to derive that **s** is used to determine if ghosts were made, but using **madeGhosts** makes it more obvious and self-documenting. I like to use more verbose variables as opposed to single letters when possible.